dhis-2/pom.xml

```diff
@@ -1653,15 +1656,26 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.jclouds.provider</groupId>
```

Comment:
Use two-space indentation for pom.xml.

Reply:
@larshelge This PR is for 2.38 where there is a four-space indentation. I have a separate PR for 2.39 with two-space indentation at [pull/10978](https://github.com/dhis2/dhis2-core/pull/10978). I've submitted this as two independent PRs rather than trying to backport and fight the indentation change. :)

(Both PRs are running into the unrelated integration test troubles that are noted in Slack.)

Reply:
Ah, it seems that dhis-2/pom.xml was changed from 2-space to 4-space [here](https://github.com/dhis2/dhis2-core/pull/8438/files#diff-4caf3d06d60c2fda63c091b5d0a1f29e447208cae834b134367e1342cf5ffb40) before 2.38 was released.

This was fixed back from 4-space to 2-space [here](https://github.com/dhis2/dhis2-core/pull/10789/files#diff-c7edcca96f2e943ecc9b19588780e38a30ed8b1f95c86796c51367b55b967968) for 2.39 but was not fixed in 2.38.

That's why they're currently different.

Reply:
Understand.